pvlib/modelchain.py

```diff
@@ -138,12 +138,12 @@ def basic_chain(times, latitude, longitude,
     airmass_model : str, default 'kastenyoung1989'
         Passed to atmosphere.relativeairmass.
 
-    altitude : None or float, default None
-        If None, computed from pressure. Assumed to be 0 m
+    altitude : float, optional
+        If not specified, computed from pressure. Assumed to be 0 m
         if pressure is also None.
```

Comment:
```suggestion
        If not specified, computed from ``pressure``. Assumed to be 0 m
        if ``pressure`` is None.
```

Reply:
I've changed `is also None` to `is also unspecified`. What do you think about it?

Reply:
[This](https://github.com/echedey-ls/pvlib-python/commit/6171e77f3831fe0184e375ee45188ebf644018dd) is the commit.